linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/bash/shared.sh

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_sle,multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ bash_ensure_pam_module_options('/etc/pam.d/common-auth', 'auth', 'required', 'pam_tally2.so', 'even_deny_root', '', '') }}}
```

Comment:
I recommend to use the `bash_ensure_pam_module_option` macro instead of the `bash_ensure_pam_module_options` (with s). Besides some issues with this macro, recently, a considerable work was done to refactor the PAM related macros, making them more robust, flexible and aligned between Ansible and Bash.

Reply:
:bow: Done in e3f3597